### ISSUE TITLE
feat:React Native SDK

### ISSUE THREAD

**Issue by phyleria** [Chimoney/chimoney-community-projects#369]
### Description

 Create a React Native SDK that allows mobile developers to easily integrate Chimoney’s services into their React Native applications. The SDK should provide access to all relevant Chimoney API functionalities such as global payouts, payment requests, etc while ensuring smooth performance on both iOS and Android platforms.

### Tasks:
- Create a wrapper around Chimoney’s API for React Native.
- Implement methods for common use cases such as sending/receiving payments, querying transactions, and user authentication.
- Ensure the SDK is compatible with both iOS and Android.
- Write documentation and provide usage examples.
- Test the SDK for common edge cases and performance.

### Ready to Work?

- [ ] I want to work on this issue

**Comment by JosephMugo8711** [comment]
I want to work on this issue @phyleria

**Comment by phyleria** [comment]
Sure, go ahead.

**Comment by vickywane** [comment]
Hi @JosephMugo8711 

This is a feature I have quite an interest in and experience working on. 

Do let me know if you want to collaborate or have difficulties working on it. 

Cc: @phyleria

**Comment by JosephMugo8711** [comment]
Hi @vickywane 
I am looking forward to it 
Here is my number  +254705049364 kindly reach me out

**Comment by JosephMugo8711** [comment]
Yes, I'm interested.  I have contacted you via your email. Nwami

**Comment by phyleria** [comment]
Amazing!!

I love the collaboration, please let me know if there's anything I can help with.

**Comment by JosephMugo8711** [comment]
Thank you and we will.

**Comment by oruly1** [comment]
## 🌟 This Issue is Now a Featured Oruly Task  sponsored by <strong>Chimoney</strong>! 🚀

Hello there! 🎉 This GitHub issue has been upgraded to a task on Oruly, where you can earn rewards for your Github contributions.

### 🚀 Task Details:
- **💡 Title:** feat:React Native SDK
- **💰 Reward:** 10 USD

### Why Oruly?
- **Diverse Task Opportunities:** Oruly offers a wide range of tasks encompassing coding, content creation, data analysis, and more, catering to various professional needs.
- **Global Collaboration:** Connect with a worldwide network of professionals. Oruly not only opens doors to global opportunities but also fosters a community where diversity and cooperative progress thrive.
- **Continuous Learning and Growth:** At Oruly, we believe in the continuous professional development of our contributors. Engage with tasks that challenge and refine your skills, supported by tools that offer insights and learning on-the-go.
- **Rewarding Contributions:** We value your time and expertise. Oruly ensures fair compensation and recognition for your work, with transparent and fast payment processes and competitive rewards.

### Claiming Your Rewards
- **Easy Process:** Complete the Github Pull Request, as normal, and submit your approved PR link on Oruly. Your reward will be automatically credited to your Oruly wallet after Approval.
- **Transparent System:** All rewards are visible on your dashboard. You can review your earned rewards anytime.
- **Quick Access:** To claim your reward, simply go to the 'Wallet' section on your profile, and select 'Payout'. Your rewards can be transferred to your bank account, mobile money, airtime or used within the platform for services or to fund your own tasks.

### Interested in Tackling This Task?
[![Claim This Task on Oruly](https://img.shields.io/badge/Claim%20This%20Task%20on%20Oruly-Claim-blue?style=for-the-badge&logo=oruly)](https://oruly.ai/task/54c3e9fd-e64e-4ef2-b004-7e84755b993f)

Transform this issue into high-quality code and contribute to exciting projects! 🚀

> "Strive not to be a success, but rather to be of value." – Albert Einstein

We're excited to see what you'll build! 🚀🌟

**Comment by vickywane** [comment]
@JosephMugo8711  are you still working on this? 

If not, I would love to take over the ticket and work on it alone. 

Cc: @phyleria

**Comment by wametee** [comment]
Hello, @phyleria 
I lost access to my GitHub account @JosephMugo8711  due to 2FA issues.
Currently, I am working from this GitHub account and continuing the development of the SDK.

**Comment by phyleria** [comment]
Hi @wametee 

Ok, noted ✅

**Comment by wametee** [comment]
@vickywane Hello,

Kindly take over the ticket and work on it.

Cc: @phyleria
